### PR TITLE
Add email service

### PR DESCRIPTION
**Pull request by Wardormeur**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -156,10 +158,26 @@ services:
     depends_on:
       - db
     volumes:
+      - ./workspace-zen/cp-translations:/usr/src/app/node_modules/cp-translations:ro
```
Comment (DanielBrierton) [inline docker-compose.yml]: Remove

File: docker-compose.yml [hunk 2]
```diff
@@ -156,10 +158,26 @@ services:
...
+    environment:
+      - EMAIL_DEFAULT_FROM='The CoderDojo Team <info@coderdojo.org>'
+      - MICROSERVICE_PORT=3000
+      - TRANSPORT=smtp
+      - SMTP_HOST=maildev
+      - TEMPLATES_DIR=/usr/src/app/node_modules/cp-translations/email-templates
+      - SMTP_SECURE=false
+      - SMTP_PORT=25
+    volumes:
+      - ./workspace-zen/services/email-service:/usr/src/app:Z
```
Comment (DanielBrierton) [inline docker-compose.yml]: Shouldn't this be `cp-email-service`? That's the repo that actually runs the service, right?

Comment (butlerx) [reply docker-compose.yml]: shoudl also be added to the setup script :smiling_imp:

Comment (DanielBrierton) [reply docker-compose.yml]: Just gotta look up at the review comment 😛 

> Add the new service to setup repos bash script.

Comment (butlerx) [reply docker-compose.yml]: nah :stuck_out_tongue_closed_eyes:

**Pull request merged**